src/libraries/System.Net.Quic/tests/FunctionalTests/MsQuicCipherSuitesPolicyTests.cs

```diff
@@ -62,11 +63,14 @@ public void NoSupportedCiphers_ThrowsArgumentException(TlsCipherSuite[] ciphers)
                     return ValueTask.FromResult(serverOptions);
                 }
             };
-            Assert.ThrowsAsync<ArgumentException>(async () => await CreateQuicListener(listenerOptions));
+            await using var listener = await CreateQuicListener(listenerOptions);
```

Comment:
How come this was supposed to be throwing before?

Reply:
Looking at this again, the QuicListener creation should not throw, because we don't check the CipherSuitesPolicy there, we only access it when we get an incoming connection.